spring-integration-core/src/main/java/org/springframework/integration/handler/DelayHandler.java

```diff
@@ -689,7 +690,7 @@ public static final class DelayedMessageWrapper implements Serializable {
 		@SuppressWarnings("serial")
 		private final Message<?> original;
 
-		DelayedMessageWrapper(Message<?> original, long requestDate) {
+		public DelayedMessageWrapper(Message<?> original, long requestDate) {
```

Comment:
Looks like solution is simpler:
```
		@JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
		DelayedMessageWrapper(@JsonProperty("original") Message<?> original, @JsonProperty("requestDate") long requestDate) {
```
Then we don't need that new `DelayedMessageWrapperJacksonDeserializer` class.